packages/features/ee/workflows/components/EventWorkflowsTab.tsx

```diff
@@ -70,9 +70,7 @@ const WorkflowListItem = (props: ItemProps) => {
         showToast(message, "error");
       }
       if (err.data?.code === "UNAUTHORIZED") {
-        // TODO: Add missing translation
-        const message = `${err.data.code}: You are not authorized to enable or disable this workflow`;
```

Comment:
Let's make this string a dynamic one that can accept err.data.code as variable e.g. like here https://github.com/calcom/cal.com/blob/94b3945ffc423ba3f5770fb7f8f16a3fbe181828/apps/web/public/static/locales/en/common.json#L1707

Reply:
sure, will make the changes